### ISSUE TITLE
Release notifications

### ISSUE THREAD

**Issue by dimejo** [contao/contao#3650]
What's the best way to receive release notifications? I'm subscribed to release notifications for this repository but never receive any. I suppose that's because there aren't any releases just tags. Is there a news feed which covers non-security releases on Contao.org?

**Comment by asaage** [comment]
I use this url: https://packagist.org/feeds/package.contao/core-bundle.rss
wich works fine.

**Comment by Toflar** [comment]
Please ask questions on one of the many support channels (https://contao.org/en/support.html). The issue tracker is here to help us maintain the software :)

**Comment by dimejo** [comment]
@asaage 
> I use this url: https://packagist.org/feeds/package.contao/core-bundle.rss
wich works fine.

Thanks for the link but I had something more userfriendly in mind.

@Toflar 
> Please ask questions on one of the many support channels (https://contao.org/en/support.html). The issue tracker is here to help us maintain the software :)

The many support channels you linked to basically boil down to Slack (which I don't use) and the Community forum. After registration (and confirmation) I was able to login but wasn't allowed to write posts as my account needed to be activated. I tried again today and can't even login because the account doesn't exist anymore.

Looks like I need to register on Slack now ...
